src/utils/cta-utils.ts

```diff
@@ -49,3 +60,165 @@ export function showAlreadyOptedInToast(
   }
   new Toast(deps, feUrl('/toastiframe', urlParams)).open();
 }
+
+/* Supports survey data tansfer, log survey events and store PPS values.*/
+export async function handleSurveyDataTransferRequest(
+  request: SurveyDataTransferRequest,
+  deps: Deps,
+  activityIframeView: ActivityIframeView,
+  configurationId: string | undefined,
```

Comment:
Seems like config id should always be available? Maybe this is an artifact from the past. Also nit `configurationId?: string`. Same with `attemptSurveyDataTransfer`.

Reply:
Done.